.github/workflows/ci.yml

```diff
@@ -1,7 +1,7 @@
 name: Regular CI Pipeline
 
 on:
-  pull_request:
```

Comment:
This should remain "pull_request" for `ci.yml` because we want it to be run only on code which is "pull request to master" for example.